src/extension.ts

```diff
@@ -18,6 +18,22 @@ const DEFAULT_DETAIL = "TabNine";
 
 export function activate(context: vscode.ExtensionContext) {
 
+  const command = 'TabNine::config';
+  const commandHandler = () => {
+    const request = tabNine.request("1.0.7", {
```

Comment:
We have a builtin command in the protocol for that. You can simply send

```
            "Configuration": {
            }
```

Reply:
done